content/en/docs/concepts/services-networking/service.md

```diff
@@ -439,7 +439,6 @@ spec:
     port: 80
     targetPort: 9376
   clusterIP: 10.0.171.239
-  loadBalancerIP: 78.11.24.19
```

Comment:
How about: also edit `146.148.47.155` (below, as an example of the ingress IP address) to use a [documentation IP address](https://content.pivotal.io/blog/domains-and-ip-addresses-reserved-for-documentation-and-why-you-should-use-them)?

Reply:
Well, its a good idea but for new users , in fact I didn't hear of it before

Reply:
Okay I checked . Actually , `146.148.47.155` is just a sample.

Reply:
@Abhik1998 Indeed. 146.148.47.155 is assigned to Google Cloud whereas (say) 192.0.2.127 is reserved for use in documentation.

Readers who misunderstand might try to connect to [146.148.47.155](http://146.148.47.155/) and find that it's a real website.

Reply:
okay
got it then should I commit a change 
ingressIP: `192.0.2.127`